CHANGELOG.md

```diff
@@ -16,7 +16,7 @@ with a focus on:
 **Zero-Trust Security Architecture**
 - X.509-based SPIFFE/SPIRE identity framework with federation support
 - Policy-based authorization framework with fine-grained access controls
-- Secure mTLS communication across all services
+- Secure X.509-based mutual TLS communication across all services
```

Comment:
lets not make changes to the previous versions of the changelog. we will cover this part in the upcoming release changelogs